Meta.svelte

```diff
@@ -3,15 +3,74 @@
   Metadata helper
 -->
 <script>
-  /** Page title */
+  /** Page title **/
   export let title: Optional<string> = undefined;
-  /** Page description */
+  /** Page description **/
   export let description: Optional<string> = undefined;
-  /** Page image */
+  /** Page image **/
   export let image: Optional<string> = undefined;
+  /** Page url **/
+  export let url: Optional<string> = undefined;
+  /** Site logo **/
+  export let logoUrl: string = url ? `${url}/logo.png` : undefined;
+  /** Search URL **/
+  export let searchUrl: Optional<string> = undefined;
+  /** Sitemap URL **/
+  export let sitemapUrl: string = url ? `${url}/sitemap.xml` : undefined;
+  /** Twitter Options **/
+  export let twitter: any = {
+    card: 'summary_large_image',
+    site: undefined,
+    creator: undefined,
+  };
+  /** OpenGraph Options **/
+  export let openGraph: any = {
+    type: undefined,
+    imageWidth: undefined,
+    imageHeight: undefined,
+    locale: 'en_US',
+    siteName: undefined,
+    imageAlt: title || undefined,
+  }
+  /** Facebook App ID **/
+  export let facebookAppId: Optional<string> = undefined;
```

Comment:
This seems out of scope for social meta? Isn't this for registered OAuth apps/pixel tracking?

Reply:
Partially, also for Facebook Insights, but yeah happy to leave this out.